app/views/DisplayPrefsView.js

```diff
@@ -0,0 +1,240 @@
+		showUnread,
+		showAvatar,
+		displayType
+	} = useSelector(state => state.sortPreferences);
+	const dispatch = useDispatch();
+
+	useEffect(() => {
+		const { navigation, isMasterDetail } = props;
+		navigation.setOptions({
+			title: I18n.t('Display'),
+			headerLeft: () => (isMasterDetail ? (
+				<HeaderButton.CloseModal navigation={navigation} testID='display-view-close' />
+			) : (
+				<HeaderButton.Drawer navigation={navigation} testID='display-view-drawer' />
+			))
+		});
+	}, []);
+
+	const setSortPreference = (param) => {
+		try {
+			dispatch(setPreference(param));
+			RocketChat.saveSortPreference(param);
+		} catch (e) {
+			logEvent(events.DP_SORT_CHANNELS_F);
+			log(e);
+		}
+	};
+
+	const sortByName = () => {
+		logEvent(events.DP_SORT_CHANNELS_BY_NAME);
+		setSortPreference({ sortBy: 'alphabetical' });
+	};
+
+	const sortByActivity = () => {
+		logEvent(events.DP_SORT_CHANNELS_BY_ACTIVITY);
+		setSortPreference({ sortBy: 'activity' });
+	};
+
+	const toggleGroupByType = () => {
+		logEvent(events.DP_GROUP_CHANNELS_BY_TYPE);
+		setSortPreference({ groupByType: !groupByType });
+	};
+
+	const toggleGroupByFavorites = () => {
+		logEvent(events.DP_GROUP_CHANNELS_BY_FAVORITE);
+		setSortPreference({ showFavorites: !showFavorites });
+	};
+
+	const toggleUnread = () => {
+		logEvent(events.DP_GROUP_CHANNELS_BY_UNREAD);
+		setSortPreference({ showUnread: !showUnread });
+	};
+
+	const toggleAvatar = () => {
+		logEvent(events.DP_TOGGLE_AVATAR);
+		setSortPreference({ showAvatar: !showAvatar });
+	};
+
+	const displayExpanded = () => {
+		logEvent(events.DP_DISPLAY_EXPANDED);
+		setSortPreference({ displayType: 'expanded' });
+	};
+
+	const displayCondensed = () => {
+		logEvent(events.DP_DISPLAY_CONDENSED);
+		setSortPreference({ displayType: 'condensed' });
+	};
+
+	const renderCheckBox = value => (
+		<List.Icon
+			name={value ? 'checkbox-checked' : 'checkbox-unchecked'}
+			color={value ? themes[theme].actionTintColor : null}
+		/>
+	);
+
+	const renderAvatarSwitch = value => (
+		<Switch
+			value={value}
+			onValueChange={() => toggleAvatar()}
+			testID='avatar-switch'
+			style={{ transform: [{ scale: 0.8 }] }}
+		/>
+	);
+
+	const renderRadio = value => (
+		<RadioButton
+			selected={!!value}
+			color={value ? themes[theme].actionTintColor : themes[theme].auxiliaryText}
+			size={ICON_SIZE}
+		/>
+	);
+
+	return (
+		<SafeAreaView>
+			<StatusBar />
+			<List.Container testID='display-view-list'>
+				<List.Section>
+					<List.Separator />
+					<List.Item
+						left={() => (
+							<Text
+								style={[styles.title, { color: themes[theme].titleText }]}
+								numberOfLines={1}
+							>{I18n.t('Display')}
+							</Text>
+						)}
+					/>
+					<List.Item
+						left={() => <List.Icon name='view-extended' />}
+						title='Expanded'
+						testID='expanded-display-pref'
+						right={() => renderRadio(displayType === 'expanded')}
+						onPress={displayExpanded}
+					/>
+					<List.Separator />
```

Comment:
You should add a `List.Separator` at the beginning and the end of each `List.Section`